### PR TITLE
Auditor portal updates

### PR DESCRIPTION
**Pull request by roman-karpovich**
Changes and bugfixes for latest inquiries.

### CODE REVIEW

File: EquiTrack/audit/models.py [hunk 1]
```diff
@@ -140,10 +148,37 @@ class Engagement(TimeStampedModel, models.Model):
         DISPLAY_STATUSES.cancelled: 'date_of_cancel'
     }
 
+    SHARED_IP_WITH_CHOICES = Choices(
```
Comment (ntrncic) [inline EquiTrack/audit/models.py]: we are already maintaining a list of agency choice in `partners.models`. Please try and use that list,  we do not want to be updating two choice lists.

File: EquiTrack/audit/migrations/0013_merge.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.10 on 2017-10-19 12:33
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```
Comment (ntrncic) [inline EquiTrack/audit/migrations/0013_merge.py]: please remove any merge migrations

Comment (ntrncic) [inline EquiTrack/audit/migrations/0013_merge.py]: please remove any merge migrations

File: EquiTrack/audit/models.py (hunk 1 above)
Comment (ntrncic) [inline EquiTrack/audit/models.py]: we are already maintaining a list of AGENCY_CHOICES in partners.models. Please try and use that list, we do not want to be updating two choice lists.

**Comment by maximlomakin** [comment]
@ntrncic Requested changes added.

**Comment by ntrncic** [comment]
@maximlomakin please resolve conflicts.

**Pull request merged**